src/coreComponents/denseLinearAlgebra/common/common.hpp

```diff
@@ -0,0 +1,46 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file common.hpp
```

Comment:
I kept the same folder structure we had in  `LinearAlgebra` but, as a matter of fact, for now it is only layouts. I guess it could contain common stuff to different `denseLinearAlgebra` interfaces but for now we only have blasLapack. We will have Magma though.